ctru-rs/src/applets/mii_selector.rs

```diff
@@ -56,14 +56,20 @@ pub struct MiiSelector {
 #[derive(Clone, Debug)]
 pub struct SelectionResult {
     pub mii_data: MiiData,
+
+    #[deprecated]
     pub is_mii_selected: bool,
```

Comment:
I suggest to simply remove the `is_mii_selected` attribute altogether. As @Techie-Pi said, it's a remnant of a 1 to 1 translation of the C API, but we don't need it here.

Also, we never released the library anyways, so there is no reason to use `#[deprecated]`. We are undergoing big API changes and it's currently vary unstable either way.

Reply:
Sure, will do. Do I need to do something to update a version number or anything?